### DISCUSSION TITLE
.net core offering

### DISCUSSION THREAD

**Discussion by vdavi** [microsoft/RulesEngine#177] [Ideas]
Any plans to move toward .net core? Allows more lightweight, flexible deployment options.

**Comment by abbasc52** [comment]
Hi @vdavi,
Can you provide more details on this? We are currently in .net standard 2.0 which is compatible with both .net framework and .net core. 

Are you looking for RulesEngine to be compiled specifically to some dotnet core version?

**Comment by ArvindLakra** [comment]
Hi, I am to compile with .net Core project. However, wanted to know if build with .dotnet core 6 is planned anytime soon ?
